### DISCUSSION TITLE
Downloading stuck

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [komikku-app/komikku#307] [General]
sometimes which happens rarely download gets stuck after completing e.g
if a manga have 13 pages downloading will be 13/13 but it won't finish completely cancelling & restarting also result the same idk if this is bug or glitch

**Comment by cuong-tran** [comment]
Check the source if all pages in that specific chapter are loaded properly or not.

**Reply by HikaruIchijyo** [reply]
it does load manually & if u forve stop app & clear cache it does download

**Reply by HikaruIchijyo** [reply]
![Screenshot_2024-10-18-09-16-34-264_app.komikku.beta.jpg](https://github.com/user-attachments/assets/a1e1aac3-bfdf-4fd2-9b28-e9ef5162c1ba)

this is what i am talking abt.
remove downloading , clear cache & restart download it fixes it
